apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -114,7 +113,7 @@ export const buildDynamicFormSchema = ({
       try {
         return JSON.parse(str);
       } catch (e) {
-        ctx.addIssue({ code: 'custom', message: 'Invalid payload. Payload needs to be a valid JSON.' });
+        ctx.addIssue({ code: 'custom', message: 'Payload needs to be a valid JSON' });
```

Comment:
Makes for easier reading with less words, also aligning "must" usage with other places.

```suggestion
        ctx.addIssue({ code: 'custom', message: 'Payload must be valid JSON' });
```